### PR TITLE
Migrate html webpack plugin to 2.8.1

### PR DESCRIPTION
**Pull request by mastertinner**
i also took the liberty of fixing some smaller imperfections

### CODE REVIEW

File: webpack.test.config.js [hunk 1]
```diff
@@ -38,30 +38,30 @@ module.exports = {
...
+        loaders: ['es6-promise', 'ts'],
         exclude: [ /\.(spec|e2e)\.ts$/ ]
       },
       {
         test: /\.ts$/,
-        loader: 'ts-loader',
+        loader: 'ts',
         query: {
-          "compilerOptions": {
-            "noEmitHelpers": true,
```
Comment (gdi2290) [inline webpack.test.config.js]: Keep these quotes

File: webpack.test.config.js [hunk 2]
```diff
@@ -22,14 +22,14 @@ module.exports = {
     preLoaders: [
       {
         test: /\.ts$/,
-        loader: 'tslint-loader',
```
Comment (gdi2290) [inline webpack.test.config.js]: I left the loader suffix because it's more clear to developers working with webpack for the first time

**Comment by mastertinner** [comment]
thanks for the feedback, @gdi2290! i changed the files accordingly.

File: webpack.prod.config.js [hunk 3]
```diff
@@ -157,7 +155,7 @@ module.exports = {
       // comments: true,//debug
 
       beautify: false,//prod
-      // disable mangling because of a bug in angular2 beta.1, beta.2, and beta.3
+      // disable mangling because of a bug in angular2 beta.1, beta.2 and beta.3
```
Comment (gdi2290) [inline webpack.prod.config.js]: no love for Oxford comma?

Comment (mastertinner) [inline webpack.prod.config.js]: As a person who's mother tongue is German, the comma before the "and" somehow doesn't look right. ;-) However, I didn't know that it was actually optional from a grammatical point of view, so sorry about correcting it.

**Pull request merged**